samples/bookinfo/src/details/start_service.sh

```diff
@@ -1,4 +1,4 @@
 
 /opt/istio/prepare_proxy.sh -p 15001 -u 1337
 ruby /opt/microservices/details.rb 9080 &
-su istio-proxy -c "/opt/istio/pilot-agent proxy -v 2 --serviceregistry Consul > /tmp/envoy.log"
+su istio-proxy -c "/opt/istio/pilot-agent proxy -v 2 --serviceregistry Consul --serviceCluster details-v1 --zipkinAddress zipkin:9411  > /tmp/envoy.log"
```

Comment:
This update allows sending traces to Zipkin.. Also, we get a nice service graph from zipkin.